setup.py

```diff
@@ -38,7 +38,15 @@
 License :: OSI Approved
 Programming Language :: C
 Programming Language :: Python
+Programming Language :: Python :: 2
+Programming Language :: Python :: 2.6
+Programming Language :: Python :: 2.7
 Programming Language :: Python :: 3
+Programming Language :: Python :: 3.2
+Programming Language :: Python :: 3.3
```

Comment:
Ops, you are right, we do test 3.2 on travis, something in the back of my head thought we stopped.